### PR TITLE
Delete cs_pg_twophase_switch_25_33.

### PR DESCRIPTION
**Pull request by ashwinstar**
Flow for test permutations:
0. Runs some pre-SQLs
1. Gets the cluster in desired state
2. pg_switch_xlog() on all the segments
3. Injects suspend fault for desired fault
4. Runs some triggers-SQLs
5. Crashes gpstop -i and start the db
6. 5 times calls pg_switch_xlog()
7. Runs some post-SQLs
8. gpcheckcat and gpcheckmirrorseg

- cluster_states: SYNC, CT, RESYNC
- fault_types: dtm_broadcast_prepare, dtm_broadcast_commit_prepared
- crash_types: gpstop_i

This seems all covered and redundant with cs_pg_twophase_switch_01_12 and
cs_pg_twophase_switch_13_24, as only diference in flow in step 6. But don't
think that makes any difference to what's being tested.

### CODE REVIEW

**Comment by Eulerizeit** [comment]
Let's get this in ASAP. Fewer CCP boxes == less likelihood of hitting the call limit.

**Comment by ashwinstar** [comment]
@jchampio have 2 commits here to fix the TINC failing on first error in CI and printing diffs. Should have ideally created separate PR for it, but lazily wish to use same branch to test it single pipeline in CI for now.

File: concourse/scripts/run_tinc_test.sh [hunk 1]
```diff
@@ -14,12 +15,24 @@ cat > ~/gpdb-env.sh << EOF
...
 
+createdb gptest
+createdb gpadmin
+
 # make fsync by default off to improve test stability
 gpconfig --skipvalidation -c fsync -v off
 gpstop -u
+gpconfig -s fsync
+
+trap look4diffs ERR
```
Comment (jchampio) [inline concourse/scripts/run_tinc_test.sh]: Moving the trap line down to right before `make` would also make sure we don't dump regression.diffs if something in `tinc_env.sh` fails, but I don't feel strongly about that.

File: concourse/scripts/run_tinc_test.sh [hunk 2]
```diff
@@ -14,12 +15,24 @@ cat > ~/gpdb-env.sh << EOF
...
+function look4diffs() {
+    find ${TINCDIR} -name *.diff -exec cat {} \; >> ${TINCDIR}/regression.diffs
```
Comment (jchampio) [inline concourse/scripts/run_tinc_test.sh]: `TINC_DIR`, I think. Setting `nounset` might be a good idea too.

File: concourse/scripts/run_tinc_test.sh [hunk 3]
```diff
@@ -14,12 +15,24 @@ cat > ~/gpdb-env.sh << EOF
...
+    echo "=================================================================="
+    echo "The differences that may have caused some tests to fail can be viewed in the file ${TINCDIR}/regression.diffs."
+    echo "=================================================================="
+    cat ${TINCDIR}/regression.diffs
```
Comment (jchampio) [inline concourse/scripts/run_tinc_test.sh]: Ditto.

**Comment by ashwinstar** [comment]
@jchampio fixed the TINC_DIR, made in consistent with `tinc_gpdb.bash`.

**Pull request merged**